usagov/vote-gov
# Vote-477 -> Yup'ik addition to testing

Link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-477

This PR will add the new Language Yup'ik to the cypress tests... This has been updated to both the home page test along with the state language tests. 

I have also updated the order of the languages in the test to reflect how they appear on the site as well. 

To test:

- pull down branch 
- run `npm-install`
- run `npm run start`
- spilt terminal 
- run `npm run cy:open`
- select both `home-page.cy.js` and `state-language.cy.js` ensure that Yup'ik is present and both tests are still passing as expected.
